### PR TITLE
Release 1.7.2

### PR DESCRIPTION
**Pull request by jdaev**
Release Notes:
- Improved error handling while uploading transaction chunks.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ dependencies:
   arweave:
     git:
       url: https://github.com/ardriveapp/arweave-dart
-      ref: 27fea131c7e5c95c7b4e0e632d9618cb6ac9d856
+      ref: v.2.0.1
```
Comment (agsuy) [inline pubspec.yaml:30]: v2.0.1

File: pubspec.lock [hunk 2]
```diff
@@ -47,11 +47,11 @@ packages:
     dependency: "direct main"
     description:
       path: "."
-      ref: "27fea131c7e5c95c7b4e0e632d9618cb6ac9d856"
-      resolved-ref: "27fea131c7e5c95c7b4e0e632d9618cb6ac9d856"
+      ref: "v.2.0.1"
```
Comment (agsuy) [inline pubspec.lock:50]: v2.0.1

**Review by agsuy — CHANGES_REQUESTED** [review]
Need to fix tag references

File: pubspec.lock (hunk 2 above)
Comment (jdaev) [reply pubspec.lock]: [Done](https://github.com/ardriveapp/ardrive-web/pull/365/commits/6010685e09748f2f7e4465e403aee38c206797cd)

**Review by agsuy — APPROVED** [review]

**Review by karlprieb — APPROVED** [review]

**Pull request merged**